jawa/util/classloader.py

```diff
@@ -73,7 +76,7 @@ def add_path(self, *paths):
             # get the index, and unpack it into our path map.
             if path.lower().endswith(('.zip', '.jar')):
                 zf = ZipFile(path, 'r')
-                self.path_map.update(izip(zf.namelist(), repeat(zf)))
+                self.path_map.update(six.zip(zf.namelist(), repeat(zf)))
```

Comment:
This should actually be `six.moves.zip`, not sure why I didn't run into this earlier.